pyspark/bigdl/contrib/onnx2bigdl/import_model.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import onnx
+from .import_graph import BigDLGraph
+
+
+class BigDLModel(object):
```

Comment:
I think this class is an utility instead of stateful class, and `BigDLModel` is confusing. `ONNXLoader` makes more sense.